WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerActivity.java

```diff
@@ -243,6 +251,13 @@ private void launchWPMediaLibrary() {
         }
     }
 
+    private void launchStockMediaPicker() {
+        if (mSite != null) {
+            ActivityLauncher.showStockMediaPickerForResult(this,
+                    mSite, RequestCodes.STOCK_MEDIA_PICKER_SINGLE_SELECT);
+        }
```

Comment:
can we add a Toast if `mSite` is null? I know such a situation is unlikely to happen, but in that event then no action is performed and the user is not informed. Maybe we can fail in the `onCreate` method in the Activity itself. Not a direct part of this PR but it seems there are several checks on `mSite` not being null that could benefit from having this check right from start.

Reply:
Added in cbbbb75, along with ensuring the stock photo menu item doesn't appear when there isn't a site.